addons/GME/Scripts/Game/GME/Editor/Modules/Reinforcements/GME_Modules_SpawnReinforcements.c

```diff
@@ -0,0 +1,160 @@
+		if (!m_pVehicle)
+			return; // To Do: Handle non-interactive case
+		
+		if (!m_pCrewGroup)
+			return; // To Do: Handle non-interactive case
+		
+		if (!m_pPassengerGroup)
+			return; // To Do: Handle non-interactive case
+				
+		m_pCrewGroup.GetOnWaypointCompleted().Insert(OnLZReached);
+		m_pCrewGroup.AddWaypoint(AIWaypoint.Cast(GetGame().SpawnEntityPrefab(Resource.Load("{3C790BB71C5CD960}PrefabsEditable/Auto/AI/Waypoints/E_AIWaypoint_ForcedMove.et"), GetWorld(), m_LZParams)));
+		m_pPassengerGroup.AddWaypoint(AIWaypoint.Cast(GetGame().SpawnEntityPrefab(Resource.Load("{FFF9518F73279473}PrefabsEditable/Auto/AI/Waypoints/E_AIWaypoint_Move.et"), GetWorld(), m_RPParams)));
+	}
+	
+	//------------------------------------------------------------------------------------------------
+	protected void OnLZReached(AIWaypoint wp)
+	{
+		GME_VehicleHelper.EjectPassengers(m_pVehicle);
+		
+		// RTB or complete module after passengers have been dropped
+		switch (m_eVehicleBehavior)
+		{
+			case GME_Reinforcements_EVehicleBehavior.STAY_AT_LZ:
+			{
+				SCR_EntityHelper.DeleteEntityAndChildren(this);
+				break;
+			}
+			
+			case GME_Reinforcements_EVehicleBehavior.RTB_AND_DESPAWN:
+			{
+				GetGame().GetCallqueue().CallLater(DoRTB, RTB_DELAY_MS);
+				break;
+			}
+		}
+				
+		m_pCrewGroup.GetOnWaypointCompleted().Remove(OnLZReached);
+	}
+	
+	//------------------------------------------------------------------------------------------------
+	protected void DoRTB()
+	{
+		m_pCrewGroup.AddWaypoint(AIWaypoint.Cast(GetGame().SpawnEntityPrefab(Resource.Load("{3C790BB71C5CD960}PrefabsEditable/Auto/AI/Waypoints/E_AIWaypoint_ForcedMove.et"), GetWorld(), m_SpawnPointParams)));
+		m_pCrewGroup.GetOnWaypointCompleted().Insert(OnRTBCompleted);
+	}
+	
+	//------------------------------------------------------------------------------------------------
+	//! Delete vehicle after RTB has been completed
+	protected void OnRTBCompleted(AIWaypoint wp)
+	{
+		SCR_EntityHelper.DeleteEntityAndChildren(m_pCrewGroup);
+		SCR_EntityHelper.DeleteEntityAndChildren(m_pVehicle);
+		SCR_EntityHelper.DeleteEntityAndChildren(this);
+	}
+	
+	//------------------------------------------------------------------------------------------------
+	void SetVehicle(Vehicle vehicle)
+	{
+		m_pVehicle = vehicle;
+	}
+	
+	//------------------------------------------------------------------------------------------------
+	Vehicle GetVehicle()
+	{
+		return m_pVehicle;
+	}
+	
+	//------------------------------------------------------------------------------------------------
+	void SetCrewGroup(SCR_AIGroup group)
+	{
+		m_pCrewGroup = group;
+	}
+	
+	//------------------------------------------------------------------------------------------------
+	SCR_AIGroup GetCrewGroup()
+	{
+		return m_pCrewGroup;
+	}
+	
+	//------------------------------------------------------------------------------------------------
+	void SetPassengerGroup(SCR_AIGroup group)
+	{
+		m_pPassengerGroup = group;
+	}
```

Comment:
So would this make it limited to only selecting a single group to add as passengers? So if you select a single individual soldier, you can only have that one, as that is a single group?

Reply:
Good point, I'll try to find a way to implement selecting multiple groups

Reply:
Would it be possible to just change to have an "array" of passenger groups instead of a single ``AIGroup``?

Reply:
Selecting multiple groups as passengers is now possible. Part of it was already implemented in #52